src/config.rs

```diff
@@ -40,12 +40,15 @@ fn true_bool() -> bool {
 #[derive(Clone, Debug, Deserialize)]
 pub struct Selection {
     pub semantic_escape_chars: String,
+    #[serde(default)]
+    pub save_to_clipboard: bool,
```

Comment:
Since this doesn't have a failure case implemented, it would completely invalidate the Alacritty configuration file if the user specifies a value like `trues`.

To resolve this the `deserialize_with = "failure_default"` annotation should be added. You should find the concrete way to do this with several other bindings in the `config.rs`.